modules/ROOT/pages/fd-connection-edit.adoc

```diff
@@ -0,0 +1,31 @@
+= Edit a Connection
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords:
+
+:imagesdir: ../assets/images
+
+You can edit a connection that is being used by or is available to be used by a specific type of connector, changing any of the information that you have specified in it.
```

Comment:
So...this seems like a lot of words to say you can edit a connection, whether in use or just available.
My head is kind of spinning already just reading sentences with multiple instances of connector, connection, and configuration. My Minimalist Me (not to be confused with my Mini Me) is kicking to get out.